values.yaml

```diff
@@ -65,6 +65,15 @@ global:
   # deployment: [codeRelease | example | <any previous code release tag>] the target deployment mode for partitioning external type dependencies, the value "codeRelease" will align it with the java parent project version
   deployment: codeRelease
 
+  # set environment variables for all palisade services pods by deployment type eg. spring profiles active
+  env:
+    _default:
+    - name: "SPRING_PROFILES_ACTIVE"
+      value: "k8s"
+    codeRelease:
+    - name: "SPRING_PROFILES_ACTIVE"
+      value: "k8s"
```

Comment:
why a _default and a code release?

Reply:
Set different profiles depending upon deploy type. By default we want to run the jars with the k8s profile, on codeRelease we also want the k8s profile, but in example deployment we want to have example-prepopulation data loaded, so need the example-libs profile AND k8s

Reply:
delete code release one then

Reply:
deleted